### PR TITLE
added repository to package.json

### PR DESCRIPTION
**Pull request by gsouf**
Hi,
Without this we cannot find the github repo from npm.

Also, if your library is stable would you mind releasing version 1.0.0 please?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "hangul-romanization",
+  "package": "Tyriar/hangul-romanization",
```
Comment (Tyriar) [inline package.json]: Isn't it better to use repository like this? https://github.com/xtermjs/xterm.js/blob/d6a45b511ecd30c841133381e355ffce483495a1/package.json#L8

**Review by Tyriar — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (gsouf) [reply package.json]: Either works, but I changed to use the full url :)

**Review by Tyriar — APPROVED** [review]

**Comment by gsouf** [comment]
@Tyriar sorry I just figured out the issue using "package" instead of repository!

**Comment by Tyriar** [comment]
np 🙂 publishing v1.0.0 now

**Comment by Tyriar** [comment]
Done, let me know if there are problems.

**Pull request merged**